src/faction_camp.cpp

```diff
@@ -1618,7 +1618,11 @@ void basecamp::abandon_camp()
     }
     overmap_buffer.remove_camp( *this );
     map &here = get_map();
-    here.remove_submap_camp( here.getlocal( bb_pos ) );
+    const tripoint &sm_pos = omt_to_sm_copy( omt_pos.raw() );
+    const tripoint &ms_pos = sm_to_ms_copy( sm_pos );
```

Comment:
These return copies, not references.
```suggestion
    const tripoint sm_pos = omt_to_sm_copy( omt_pos.raw() );
    const tripoint ms_pos = sm_to_ms_copy( sm_pos );
```

Reply:
I was binding the return value from these two functions to local constant references.